test/themes/base.theme.svn.bats

```diff
@@ -120,6 +123,8 @@ function setup_svn_path {
 
   setup_svn_path "$BASH_IT/test/fixtures/svn/broken"
 
+  unset SVN_EXE
```

Comment:
Its pretty weird that we need to unset this in order for this to work..
I would prefer if we will override the value of `SVN_EXE` and the rest of them each time we source `base.theme`

Reply:
This is an artifact of trying to apply my changes against the master branch...and being lazy with the tests. I pushed a better version, but broke the tests again. I'm starting to get the hang of BATS but I just need to get a better feel for it's behavior. (BATS basically runs with `set -e`, but that's *hardcore* and I need to update my habits to account for it. This is good! It's why we have tests! My brain is stubborn!)

Reply:
Looks like I need to [not use `run` as it causes a subshell]( https://bats-core.readthedocs.io/en/stable/writing-tests.html ), as well as account for `set -e`...

Ok, eighth time's the charm!